portfolio/src/main/java/com/google/sps/servlets/RandomResponseServlet.java

```diff
@@ -0,0 +1,55 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Handles requests sent to the /hello URL. Try running a server and navigating
+ * to /hello!
+ */
+@WebServlet("/rand_response")
+public class RandomResponseServlet extends HttpServlet {
+
+    /** Random set of messages gathered from google and 
+     * https://bestlifeonline.com/world-facts/. */
+    private String[] randMessages = 
```

Comment:
If this is not going to change, consider making it immutable with the final keyword.